src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -69,6 +69,8 @@ namespace QgsWfs
 
     QString createFeatureGeoJSON( const QgsFeature &feature, const createFeatureParams &params, const QgsAttributeList &pkAttributes );
 
+    QDomElement createFieldElement( const QgsField &field, const int idx, const QgsAttributes &featureAttributes, QDomDocument &doc );
```

Comment:
Could we use a `QVariant` in parameter instead of `QgsAttributes` + the corresponding index?

Reply:
And I'm in favor of the `const int` notation but only in the implementation part, not in the definition (it may be confusing from an API point of view).